libmscore/lyrics.h

```diff
@@ -82,8 +82,9 @@ class Lyrics final : public TextBase {
       int subtype() const override            { return _no; }
       QString subtypeName() const override    { return QObject::tr("Verse %1").arg(_no + 1); }
       void setNo(int n)                               { _no = n; }
+      void setEvenLyricStyle(bool b);
       int no() const                                  { return _no; }
-      bool isEven() const                             { return _no % 1; }
+      bool isEven() const                             { return _no & 1; }
```

Comment:
See also #7628 and #7627

Reply:
1600 PRs later and at least a part of this has been merged

Reply:
And then even only that single line :-/